src/LibHdb++MySQL.h

```diff
@@ -21,7 +21,7 @@
 #define _HDBPP_MYSQL_H
 
 #include <mysql.h>
-#include "LibHdb++.h"
+#include <libhdb++/LibHdb++.h>
```

Comment:
I agree including with the path libhdb++/ is necessary for the debian packaging, but to allow compiling against the libhdb++ under development, should we move LibHdb++.h from src/ to include/libhdb++/ or what else you suggest?

Reply:
I have either setup a local install point or (since I worked mainly in docker images after I had the header nightmare), simply installled libhdbpp into the system so its under /usr/local/etc.

For the local install, I used an extra directory in my workspace home/sjames/build and installed all hdb++ libraries (and other support libraries) to there. Done using -DCMAKE_INSTALL_PREFIX=/home/sjames/build. Then I simply passed this location to the builds, i.e. set LIBHDBPP_INC=/home/sjames/build/include or -DCMAKE_INCLUDE_PATH=/home/sjames/buid/include. Of course, you need to update the LD_LIBRARY_PATH for the linker, so it picks up the shared libs when you run the device servers.

There is no reason why we can not install libhdbpp directly into its own directory using the above. I build the install rules for each mobule to have a high level of flexibility, so in theory we can redirect it just about anywhere.

I am going to add some information about this to my documentation list. While its obvious to me (having made the changes myself), an outlining for getting a complete development environment up would be a real plus. libhdbpp INSTALL.md does detail all the flags and building the lib etc, but perhaps a description/example of a complete development setup would be good also?

I can add this to the readthedocs source and point the various README.md's at it.